services/requester/requester.go

```diff
@@ -319,3 +366,82 @@ func bytesFromCadenceArray(value cadence.Value) ([]byte, error) {
 
 	return res, nil
 }
+
+func getErrorForCode(errorCode uint64) error {
```

Comment:
Maybe worth adding a todo that this would make sense to live inside flow-go/evm.

Reply:
Added TODO in https://github.com/onflow/flow-evm-gateway/pull/81/commits/fac85cb67c8431ed48f45c170d5ecfbd9c1fb2cf